Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -239,18 +238,51 @@ export class ProjectDetailsComponent {
         };
       }))
     ]).subscribe((res: [GenericResponseDTO, GenericResponseDTO]) => {
-      if(res[0].success == true && res[1].success == true){
+      if (res[0].success == true && res[1].success == true) {
         this.details.description = this.updateProjectForm.get("details").value;
         this.details.tags = this.updateProjectForm.get("tags").value;
         this.toastrService.success("The project has been saved successfully", "Project Saved");
-      }else{
+      } else {
         this.toastrService.danger("There was an error updating the project", "Error");
       }
       this.loadingProject = false;
     }, (err) => {
       this.toastrService.danger("There was an error updating the project", "Error");
       this.loadingProject = false;
     })
+    this.pageMode = 'view';
+  }
+
+  removeUserCheck(removeUserId: number) {
+    let title = "Are you sure you want to remove this user?";
+    let body = `Removing this user from the project will also remove all existing
+     time entries the user has created on this project.`;
+    this.dialogService.open(YesNoDialogComponent, {
+      context: {
+        body: body,
+        title: title,
+        yesClicked: () => {
+          this.removeUser(removeUserId)
+        }
+      }
+    });
+  }
+
+  removeUser(removeUserId: number) {
+    let removeUserInfo: ProjectRemoveUserDTO = {
+      projectId: this.details.id,
+      userId: removeUserId
+    }
+    this.projectService.removeUserFromProject(removeUserInfo).subscribe((res: GenericResponseDTO) => {
+      if (res.success) {
+        this.toastrService.show("Success", "User Id: " + res.data + " successfully removed from your project", { status: 'success', duration: 4000 });
+        setTimeout(() => {  window.location.reload(); }, 2000);
```

Comment:
Reloading the page is a bad user experience, instead just remove that row from the data set as you know at this point it was successfully completed.